server/tests/api_v4/authenticated/test_shamir_recovery_setup.py

```diff
@@ -85,46 +86,49 @@ async def test_authenticated_shamir_recovery_setup_share_inconsistent_timestamp(
     assert rep == authenticated_cmds.v4.shamir_recovery_setup.RepShareInconsistentTimestamp()
 
 
+@pytest.mark.xfail(
+    reason="TODO: currently there is a unique shamir topic, we should switch to a per-user shamir topic instead"
+)
```

Comment:
Is that still necessary?

Reply:
Yes because currently the memory implementation of shamir in server has a single shamir topic for the whole organization (not a shamir topic per-user)